<?php 
  require "html/header.php";
?>


	<div class="two-cols-even">
		<div class="col1 box1">
			<!--<h2 class="title">Research that we focus on</h2>-->
			<h2 class="title">Identification of alternative splicing using PacBio/Nanopore</h2>
			<div class="entry">
				<ul class="list1">
					<li class="first">
						<p><img src="<?php echo $config_domain;?>/html/images/Fig.1.png" width="194" height="328" alt="" class="alignleft" /> We are pleasure to release out first version of PRAPI, which is one-stop solution for Iso-Seq analysis of analyze alternative transcription initiation (ATI), alternative splicing (AS), alternative cleavage and polyadenylation (APA), natural antisense transcripts (NAT), and circular RNAs (circRNAs) comprehensively. We also developed methods to identify differential AS events for species without reference genome. </p>						<p><a href="<?php echo $config_domain;?>/html/research.php" class="link1">Read more...</a></p>
					</li>
				

					<h2 class="title">The function of circular RNA</h2>
					<div class="entry">
					<li>
						<p><img src="<?php echo $config_domain;?>/html/images/Fig8.png" width="194" height="218" alt="" class="alignleft" /> We used circular RNA sequencing to identify the function of circular RNA in plant. We recently started to performance molecular experiment to investigate circular RNA on gene expression and post-transcription regulation.</p>
						<p><a href="<?php echo $config_domain;?>/html/research.php" class="link1">Read more...</a></p>
					</li>
		
				</ul>
			</div>
		</div>



		<div class="col2">

                        <div class="box1">
                                <h2 class="title">Accepted papers</h2>
                                <div class="entry">
					<ul class="list2">
<li><strong>2019/1</strong><a href="https://academic.oup.com/bioinformatics/article-abstract/35/17/3119/5299997?redirectedFrom=fulltext">Bioinformatics.</a></li>
<li><strong>2018/12</strong><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/tpj.14159">The plant journal.</a></li>
<li><strong>2017/12</strong><a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btx830/4769502?redirectedFrom=fulltext">Bioinformatics.</a></li>
<li><strong>2017/05</strong><a href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13597/full">The plant journal</a></li>

<!--<li><strong>2016/12</strong><a href="http://onlinelibrary.wiley.com/doi/10.1111/php.12680/abstract">Photochem Photobiol.</a></li>-->
<!--<li><strong>2016/05</strong><a href="http://www.nature.com/ng/journal/v48/n6/full/ng.3555.html">nature genetics</a></li>-->
<!--<li><strong>2016/05</strong><a href="http://www.pnas.org/content/113/19/5447.long">Proceedings of the National Academy of Sciences</a></li>-->
<!--<li><strong>2016/03</strong><a href="http://www.nature.com/articles/srep23173">Sci. Rep.</a></li>-->
<!--<li><strong>2016/01</strong><a href="http://www.plantcell.org/content/28/1/55.abstract">plant Cell</a></li>-->


                                                <!--<li><strong>06/2015</strong><a href="http://www.nature.com/cr/journal/v25/n7/full/cr201577a.html">Cell Research</a></li> -->
                                        <!--        <li><strong>07/2014</strong><a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0106001&representation=PDF">PLOS ONE</a></li> -->
                                                <!--<li><strong>01/2014</strong><a href="http://www.pnas.org/content/111/10/3877.short">Proceedings of the National Academy of Sciences</a></li> -->
                                                <!--<li><strong>01/2013</strong> <a href="http://www.plantcell.org/content/25/2/715.short">The Plant Cell </a></li> -->
                                                <!--<li><strong>03/2010</strong> <a href="http://www.pnas.org/content/107/44/19114.short">Proceedings of the National Academy of Sciences</a></li> -->
                                                <!--<li><strong>11/2009</strong> <a href="http://link.springer.com/article/10.1007/s11515-010-0007-8">Frontiers in Biology</a></li> -->
                                                <!-- <li><strong>08/2006</strong> <a href="http://www.sciencedirect.com/science/article/pii/S1673852707600265">Journal of Genetics and Genomics</a></li> -->
                                        </ul>
					<p><a href="http://scholar.google.com/citations?user=jnrDUeQAAAAJ&hl=en" class="link1">Google Scholar</a> </p>
                                </div>
                        </div>


                                <!--<h2 class="title">Software</h2> -->
                                <!--<div class="entry"> <img src="<?php echo $config_domain;?>/html/images/PLOSas.png" width="191" height="171" alt="" class="alignleft" /> -->
                                <!--        <p>We developed one method which did not dependent on the cDNA-to-genome alignment to detect alternative splicing. The method can detect alternative splicing by RNA-seq for species without a reference genome. The method has been published. At the same time we will release another software to detect alternative splicing and differential splicing events according to cDNA-to-genome alignment method.</p>-->
                                <!--</div>-->



			<div class="box1">
				<h2 class="title">Database</h2>
                                <div class="entry"> <a href="http://forestry.fafu.edu.cn/bioinfor/db/ASmiR/" class="link1"> <img src="<?php echo $config_domain;?>/html/images/ASmiRdb.png" width="198" height="196" alt="" class="alignleft" /> ASmiR databases</a>


                                        <p>
We are pleasure to release out first version of ASmiRdb: the interplay between miRNA and alternative splicing in plant. The database was accompanied by easy-to-use web query interfaces for data visualization and downstream analysis. Especially, the forest genomics community also can submit own dataset to the web service to search for the small RNA target sites locating at AS region. </p>
					    <!--	<a href="http://www.bioinfor.org/gb/TAIR10/?loc=chr01%3A3994376..3999145&tracks=TAIR10&highlight=" class="link1">ath</a> -->					
	
					     <!-- <a href="http://www.bioinfor.org/gb/TIGR7/?loc=chr08%3A11603661..11613200&tracks=DNA%2CTransposon%2CTIGR7&highlight=" class="link1">osa</a> -->
						<!--	<a href="http://www.bioinfor.org/gb/csa/?loc=scaffold31054%3A7134..10313&tracks=canSat3_transcriptome_full%2CcanSat3_transcriptome_representative%2Cfinola1_transcriptome_representative%2Cfinola1_transcriptome_full&highlight=" class="link1">csa</a> -->

						 <!-- <a href="http://www.bioinfor.org/gb/hg19/" class="link1">hg19</a> -->

						<!-- <a href="http://www.bioinfor.org/gb/pombe/" class="link1">pome</a> -->

					<!--	<a href="http://www.bioinfor.org/gb/gra/?loc=KB204119.1%3A835206..839075&tracks=RNA-seq-assembly-Gene&highlight=" class="link1">gra</a> -->

                                       <!--  <p>The genome browser for several species is available. Local BAM, BigWig and GFF3 files can be opened without the need to transfer any data over the network by using the "File - > open function". We will load our published high throughput sequencing dataset into the browser in future.Other species will be added soon. </p> -->
                               </div>

			</div>

                                <h2 class="title"><a href="http://www.bioinfor.org/bioinfor/tool/PRAPI/" class="link1">Software</a></h2>
                                <div class="entry"><a href="http://www.bioinfor.org/bioinfor/tool/PRAPI/" class="link1"> <img src="<?php echo $config_domain;?>/html/images/PLOSas.png" width="191" height="171" alt="" class="alignleft" /> </a>
					<p>We are pleasure to release out first version of PRAPI, which a one-stop solution for Iso-Seq analysis of analyze alternative transcription initiation (ATI), alternative splicing (AS), alternative cleavage and polyadenylation (APA), natural antisense transcripts (NAT), and circular RNAs (circRNAs) comprehensively.</p>

                                </div>




		</div>
	</div>
</div>

<?php 
  require "html/footer.php";
?>

