# **Welcome to Gu LAB**

## **Description**

## **Publicaiton**

### **Paper 2022**

Yubang Gao, Xuqing Liu, Yandong Jin, Ji Wu, Shuang Li, Yaxing Li, Binqing Chen, Yaxin Zhang, Linxiao Wei, Wei Li, Ruili Li, Chentao Lin, Anireddy S N Reddy, Pankaj Jaiswal, **Lianfeng Gu**. (2022). Drought induces epitranscriptome and proteome changes in stem-differentiating xylem of Populus trichocarpa. _**Plant Physiology**_.
[doi:10.1093/plphys/kiac272](https://academic.oup.com/plphys/article/190/1/459/6603714)

Yushan Zheng, Deming Yang, Jundong Rong, Liguang Chen, Qiang Zhu, Tianyou He, Lingyan Chen, Jing Ye, Lili Fan, Yubang Gao, Hangxiao Zhang, **Lianfeng Gu**. (2022). Allele-aware chromosome-scale assembly of the allopolyploid genome of hexaploid Ma Bamboo (Dendrocalamus latiflorus Munro). _**J Integr Plant Biol.**_
[DOI: 10.1111/jipb.13217](https://onlinelibrary.wiley.com/doi/abs/10.1111/jipb.13217)

Zekun Zhang, Huiyuan Wang, Ji Wu, Yandong Jin, Shengwu Xiao, Tao Li, Xuqinq Liu,Hangxiao Zhang, Zeyu Zhang, Jun Su, Jingzao Liu, Xiaoyan Wang, Yubang Gao, Xiangqing Ma and **Lianfeng Gu**. (2022). Comprehensive Transcriptome Analysis of Stem-Differentiating Xylem Upon Compression Stress in Cunninghamia Lanceolata. **_Frontiers in genetics 13_**
[DOI:10.3389/fgene.2022.843269](https://www.frontiersin.org/articles/10.3389/fgene.2022.843269/full)

Yaxing Zhang, Jun Zhang, Deming Yang, Yandong Jin, Xuqing Liu, Zeyu Zhang, **Lianfeng Gu**, Hangxiao Zhang. (2022). Genome-wide identification of NAP1 and function analysis in moso bamboo (Phyllostachys edulis) _**International Journal of Molecular Sciences**_. Accept

Liufu, Y., Liu, X., Wu, L., Zhang, H., Gao, Y. and **Gu, L**. (2022). An Analysis Pipeline for Identification of RNA Modification, Alternative Splicing and Polyadenylation Using Third Generation Sequencing. _**Bio-101: e4433.**_
[DOI:10.21769/BioProtoc.4433.](https://doi.org/10.21769/BioProtoc.4433)



### **Papers 2021 (*first-author #corresponding author)**

Yubang Gao*, Xuqing Liu*, Bizhi Wu, Huihui Wang, Feihu Xi, Markus V. Kohnen, Anireddy S. N. Reddy, Lianfeng Gu#. (2021). Quantitative profiling of N6-methyladenosine at single-base resolution in stem-differentiating xylem of Populus trichocarpa using Nanopore direct RNA sequencing. _**Genome Biology**_
[DOI:10.1186/s13059-020-02241-7](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02241-7)

Zeyu Zhang*, Huihui Wang*, Yongsheng Wang*, Feihu Xi*, Huiyuan Wang, Markus V. Kohnen, Pengfei Gao, Wentao Wei, Kai Chen, Xuqing Liu, Yubang Gao, Ximei Han, Kaiqiang Hu, Hangxiao Zhang, Qiang Zhu, Yushan Zheng, Bo Liu, Ayaz Ahmad, Yau-Heiu Hsu, Steven E. Jacobsen#, **Lianfeng Gu#**. (2021). Whole genome characterization of chronological age-associated changes in methylome and circular RNAs in moso bamboo (Phyllostachys edulis) from vegetative to floral growth. _**Plant Journal**_
[DOI:10.1111/tpj.15174](https://onlinelibrary.wiley.com/doi/10.1111/tpj.15174)

Xuqing Liu*, Yubang Gao*, Jiakai Liao, Miao Miao, Kai Chen, Feihu Xi, Wentao Wei, Huihui Wang, Yongsheng Wang, Xi Xu, Anireddy S.N. Reddy, and **Lianfeng Gu*** . (2021). Genome-wide profiling of circular RNAs, alternative splicing, and R-loops in stem-differentiating xylem of Populus trichocarpa. _**Journal of Integrative Plant Biology**_
[DOI:10.1111/jipb.13081](https://doi.org/10.1111/jipb.13081)

Xiaqin Wang, Xinyang Yan, Shubin Li, Yun Jing, **Lianfeng Gu**, Shuangquan Zou, Jin Zhang*, Bobin Liu*, (2021), Genome-wide identification, evolution and expression analysis of the aspartic protease gene family during rapid growth of moso bamboo (Phyllostachys edulis) shoots. _**BMC Genomics25**_
[DOI:10.1186/s12864-020-07290-7](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-07290-7)

Jian Wang*, Bizhi Wu*, Markus V. Kohnen*, Daqi Lin,Changcai Yang,Xiaowei Wang,Ailing Qiang, Wei Liu,Jianbin Kang, Hua Li,Jing Shen,Tianhao Yao, Jun Su*,Bangyu Li*,and **Lianfeng Gu***. (2021). Classification of Rice Yield Using UAV-Based Hyperspectral Imagery and Lodging Feature. _**Plant Phenomics**_
[DOI:10.34133/2021/9765952](https://spj.sciencemag.org/journals/plantphenomics/2021/9765952/)

Huiyuan Wang, Sheng Liu, Xiufang Dai, Yongkang Yang, Yunjun Luo,  Yubang Gao, Xuqing Liu, Wentao Wei, Huihui Wang, Xi Xu,  Anireddy S. Reddy,  Pankaj Jaiswal,  Wei Li,  Bo Liu* and  **Lianfeng Gu***(2021). PSDX: A comprehensive multi-omics association database of Populus trichocarpa with a focus on the secondary growth in response to stresses Front._**Plant Sci**_.
[DOI:10.3389/fpls.2021.655565](https://www.frontiersin.org/articles/10.3389/fpls.2021.655565)

Guihua Jin, Peng-Fei Ma, Xiaopei Wu, **Lianfeng Gu**, Manyuan Long#, Chengjun Zhang#, De-Zhu Li#. New Genes Interacted With Recent Whole-Genome Duplicates in the Fast Stem Growth of Bamboos. (2021) _**Molecular Biology and Evolution**_
[DOI:38(12):5752–5768](https://academic.oup.com/mbe/article/38/12/5752/6377341)


Wang, Xu *, Bochen Jiang*, **Lianfeng Gu***, Yadi Chen, Manuel Mora, Mulangma Zhu, Eliace Noory, Qin Wang#, and Chentao Lin#. "A photoregulatory mechanism of the circadian clock in Arabidopsis." _**Nature Plants**_ 7 pages1397–1408 (2021)
[DOI:10.1038/s41477-021-01002-z](https://www.nature.com/articles/s41477-021-01002-z#citeas)

**Gu L**., Zhang H., Gao J. (2021) Alternative Splicing and Alternative Polyadenylation in Moso Bamboo. In: Gao J. (eds). The Moso Bamboo Genome. Compendium of Plant Genomes. _**Springer**_
[DOI:10.1007/978-3-030-80836-5_7](https://link.springer.com/chapter/10.1007/978-3-030-80836-5_7)





### **Papers 2020 (*first-author #corresponding author)**

Yongsheng*,Wang  Huihui Wang*, Feihu Xi, Huiyuan Wang,  Ximei Han,  Wentao Wei,  Hangxiao Zhang,  Qianyue Zhang,  Yushan Zheng,  Qiang Zhu,  Markus V. Kohnen,  Anireddy S. N. Reddy, **Lianfeng Gu #**. (2020). Profiling of circular RNA N6‐methyladenosine in moso bamboo (Phyllostachys edulis ) using nanopore‐based direct RNA sequencing. _**Journal of Integrative Plant Biology**_ 62(12), 1823-1838.
[DOI:10.1111/jipb.13002](https://onlinelibrary.wiley.com/doi/abs/10.1111/jipb.13002)


Reddy, A. S., Huang, J., Syed, N. H., Ben-Hur, A., Dong, S., **Gu, L**. (2020). Decoding co-/post-transcriptional complexities of plant transcriptomes and epitranscriptome using next-generation sequencing technologies. Biochemical Society Transactions, _**Biochemical Society Transactions**_
[DOI:10.1042/BST20190492](https://portlandpress.com/biochemsoctrans/article-abstract/48/6/2399/226973/Decoding-co-post-transcriptional-complexities-of)


Shanwen Ye,  Gang Chen,  Markus V. Kohnen,  Wenjia Wang,  Changyang Cai,  WenSha Ding,  Chu Wu, ** Lianfeng Gu**,  Yushan Zheng,  Xiangqing Ma,  Chentao Lin, Qiang Zhu# . Robust CRISPR/Cas9 mediated genome editing and its application in manipulating plant height in the first generation of hexaploid Ma bamboo (Dendrocalamus latiflorus Munro). _**Plant biotechnology journal**_
[DOI:10.1111/pbi.13320](https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.13320)


Jie Huang, Xinyu Lu, Hongwei Wu, Yuchen Xie, Qian Peng, **Lianfeng Gu**, Juyou Wu, Yuanchao Wang, Anireddy S.N.Reddy, Suomeng Dong#. Phytophthora Effectors Modulate Genome-wide Alternative Splicing of Host mRNAs to Reprogram Plant Immunity. _**Molecular Plant**_
[DOI:10.1016/j.molp.2020.07.007](https://www.sciencedirect.com/science/article/abs/pii/S1674205220302203)


Zhe Zhang, Xuelian Yang, Ling Cheng, Zejun Guo, Huiyuan Wang, Weihuang Wu, Kihye Shin, Jinyao Zhu, Xiaoli Zheng, Jianghu Bian, Yangchen Li,**Lianfeng Gu**, Qiang Zhu, Zhi-Yong Wang & Wenfei Wang #. Physiological and transcriptomic analyses of brassinosteroid function in moso bamboo (Phyllostachys edulis) seedlings. _**Planta**_, 252(2), 1-13.
[DOI:10.1007/s00425-020-03432-z](https://link.springer.com/article/10.1007/s00425-020-03432-z)



### **Papers 2019 (*first-author #corresponding author)**

Wang HY*, Wang HH*, Zhang H*, Liu S, Wang Y, Gao Y, Xi F, Zhao L, Liu B, Reddy A, Lin C, **Gu L#**. (2019). The interplay between microRNA and alternative splicing of linear and circular RNAs in eleven plant species _**Bioinformatics**_
[DOI:10.1093/bioinformatics/btz038](https://doi.org/10.1093/bioinformatics/btz038)

Wang Y*, Gao Y*, Zhang H*, Wang H, Liu X, Xu X, Zhang Z, Kohnen M, Hu K, Wang H, Xi F, Zhao L, Lin C, **Gu L#**. (2019). Genome-wide profiling of circular RNAs in the rapidly growing shoots of moso bamboo (Phyllostachys edulis). _**Plant and Cell Physiology**_
[DOI:10.1093/pcp/pcz043](https://academic.oup.com/pcp/advance-article-abstract/doi/10.1093/pcp/pcz043/5369732)

Yu X, Wang Y, Kohnen M, Piao M, Tu M, Gao Y, Lin C, Zuo Z#, **Gu L#**. (2019). Large Scale Profiling of Protein Isoforms Using Label-Free Quantitative Proteomics Revealed the Regulation of Nonsense-Mediated Decay in Moso Bamboo (Phyllostachys edulis). _**Cells**_ 8(7), 744. Special Issue "Alternative Splicing: Recent Insights into Mechanisms and Functional Roles"
[DOI:10.3390/cells8070744](https://www.mdpi.com/2073-4409/8/7/744)

Zhao L, Zhang H, Kohnen M, Prasad K, **Gu L#**, Reddy A#. (2019). Analysis of transcriptome and epitranscriptome in plants using PacBio Iso-Seq and Nanopore-based direct RNA sequencing _**Front. Genet**_. Research Topic: Transcript Profiling Using Single Molecule, Long-Read Sequencing Technologies.
[DOI:10.3389/fgene.2019.00253](https://www.frontiersin.org/articles/10.3389/fgene.2019.00253/full)

Wu B, Zhang H, Lin L, Wang H, Gao Y, Zhao L, Chen YP #, Chen R #, **Gu L#** (2019). A similarity searching system for biological phenotype images using deep convolutional encoder-decoder architecture. _**Current Bioinformatics**_
[DOI:10.2174/1574893614666190204150109](http://www.eurekaselect.com/169619)

Gao Y, Xi F, Zhang H, Liu X, Wang H, Zhao L, Reddy A, **Gu L #** (2019). Single-molecule real-time (SMRT) isoform sequencing (Iso-Seq) in plants: The status of the bioinformatics tools to unravel the transcriptome complexity. _**Current Bioinformatics**_
[DOI:10.2174/1574893614666190204151746](https://www.ingentaconnect.com/content/ben/cbio/2019/00000014/00000007/art00004)

Clark S, Yu F, **Gu L**, Min X# (2019). Expanding alternative splicing identification by integrating multiple sources of transcription data in tomato. _**Front.Plant Sci**_. Research Topic: Alternative Splicing Regulation in Plants.
[DOI:10.3389/fpls.2019.00689](https://www.frontiersin.org/articles/10.3389/fpls.2019.00689/full)

Liu Xu, Gao Y, Zhao L, Cai Y, Wang H, Miao M, **Gu L#**, Zhang H# (2019). Biogenesis, research methods, and functions of circular RNAs. _**Hereditas**_. Vol. 41 Issue (6): 469-485
[DOI:10.16288/j.yczz.19-061](http://www.chinagene.cn/CN/10.16288/j.yczz.19-061)

Yang Z, Chen L, Kohnen MV, Xiong B, Zhen X, Liao J, Oka Y, Zhu Q, **Gu L**, Lin C#, Liu B#. Identification and Characterization of the PEBP Family Genes in Moso Bamboo (Phyllostachys heterocycla). _**Scientific Reports**_ 2019 Oct 18;9(1):1-2 
 [DOI:10.1038/s41598-019-51278-7](https://www.nature.com/articles/s41598-019-51278-7)
 
Zhu, Danling, Fei Mao, Yuanchun Tian, Xiaoya Lin, **Lianfeng Gu**, Hongya Gu, Li-jia Qu, Yufeng Wu, and Zhe Wu. The features and regulation of co-transcriptional splicing in Arabidopsis. _**Molecular plant**_ (2019).
[DOI:10.1016/j.molp.2019.11.004](https://www.sciencedirect.com/science/article/abs/pii/S1674205219303685)



### **Papers 2018 (*first-author   #corresponding author)**

Ye G*, Zhang H*, Chen B, Nie S, Liu H, Gao W, Wang H, Gao Y, **Gu L#**. De novo genome assembly of the stress tolerant forest species Casuarina equisetifolia provides insight into secondary growth (2018). _**Plant Journal**_.
[DOI:/10.1111/tpj.14159](https://onlinelibrary.wiley.com/doi/pdf/10.1111/tpj.1415)

Gao Y, Wang H, Zhang H, Wang Y, Chen J, **Gu L#**, (2018), PRAPI: post-transcriptional regulation analysis pipeline for Iso-Seq. _**Bioinformatics**_
[DOI:10.1093/bioinformatics/btx83](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btx830/4769502)

Zhang H*, Wang H*, Zhu Q*, Gao Y, Wang H, Zhao L, Wang Y, Xi F, Wang W, Yang Y, Lin C, **Gu L#** (2018), Transcriptome characterization of moso bamboo (Phyllostachys edulis) seedlings in response to exogenous gibberellin applications, _**BMC plant biology**_, 2018,18:125
[DOI:10.1186/s12870-018-1336-z](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-018-1336-z)

Wang Q, Zuo Z, Wang X, Liu Q, **Gu L**, Oka Y, Lin C (2018), Beyond the photocycle – how cryptochromes regulate photoresponses in plants?, _**Current Option in Plant Biology**_, 45, 120-126.
[DOI:10.1016/j.pbi.2018.05.014](https://www.sciencedirect.com/science/article/pii/S1369526618300360)

Wang, G., Sun, P., Gong, Z., **Gu, L**., Lou, Y., Fang, W., Zhang, L., Su, L., Yang, T., Wang, B. and Zhou, J., 2018. Srk1 kinase, a SR protein‐specific kinase, is important for sexual reproduction, plant infection and pre‐mRNA processing in Fusarium graminearum. _**Environmental microbiology**_
[DOI:10.1111/1462-2920.14299](https://onlinelibrary.wiley.com/doi/pdf/10.1111/1462-2920.14299)

Liu B., Zhang J., Yang Z., Matsui A., Li S., Yan X., Kohnena M., **Gu L**., Prasad K., Tuskan G., Lu M.,# Oka Y.# (2018). PtWOX11 acts as master regulator conducting the expression of key transcription factors to induce de novo shoot organogenesis in poplar. _**Plant Molecular Biology**_. (2018): 1-18.
[DOI:10.1007/s11103-018-0786-x](https://dx.doi.org/10.1007/s11103-018-0786-x)



### **Papers 2017 (*first-author   #corresponding author)**

Wang T*, Wang H*, Cai D*, Gao Y, Zhang H, Wang Y, Lin C, Ma L#, **Gu L#**, (2017), Comprehensive profiling of rhizome-associated alternative splicing and alternative polyadenylation in moso bamboo (Phyllostachys edulis). _**The Plant Journal**_. 2017;91(4):684-99.
[DOI:10.1111/tpj.13597](http://onlinelibrary.wiley.com/doi/10.1111/tpj.13597/ful)


Huang J, **Gu L**, Zhang Y, Yan T, Kong G, Kong L, Guo B, Qiu M, Wang Y, Jing M, Xing W, Ye W, Wu Z, Zhang Z, Zheng X, Gijzen M, Wang Y, Dong S(2017), An oomycete plant pathogen reprograms host pre-mRNA splicing to subvert immunity. _**Nature Communications**_.
[DOI:10.1038/s41467-017-02233-5](https://www.nature.com/articles/s41467-017-02233-5#Sec10)

Ma X, Liu C, **Gu L**, Mo B, Cao X, Chen X. (2017), TarHunter, a tool for predicting conserved microRNA targets and target mimics in plants. _**Bioinformatics**_.
[DOI:10.1093/bioinformatics/btx797](https://academic.oup.com/bioinformatics/article/34/9/1574/4721784)

Wang W*, **Gu L***, Ye S, Zhang H, Cai C, Xiang M, Gao Y, Wang Q, Lin C, Zhu Q#, (2017), Genome-wide analysis and transcriptomic profiling of the auxin biosynthesis, transport and signaling family genes in moso bamboo (Phyllostachys heterocycla). _**BMC Genomics**_. 2017 Nov 13;18(1):870
[DOI:10.1186/s12864-017-4250-0](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4250-0)

Wang K*, Jiao Z*, Xu M*, Wang Y, Li R, Cui X, **Gu L#**, Zhang S#. (2017), Landscape and Fruit Developmental Regulation of Alternative Splicing in Tomato by Genome-wide Analysis. _**Horticultural Plant Journal**_.
[DOI:10.1016/j.hpj.2017.01.007](http://www.sciencedirect.com/science/article/pii/S2468014117300080)

Wang, X., Wang, Q., Han, Y.J., Liu, Q., **Gu, L**., Yang, Z., Su, J., Liu, B., Zuo, Z., and He, W., Wang,J., Liu, B., Matsui, M., Kim,J., Oka, Y., Lin,C (2017). A CRY-BIC negative feedback circuitry regulating blue light sensitivity of Arabidopsis. _**The Plant Journal**_.
[DOI:10.1111/tpj.13664](http://onlinelibrary.wiley.com/doi/10.1111/tpj.13664/full)

Hansheng Zhao*, Shancen Zhao*, International Network for Bamboo and Rattan, Benhua Fei, Huan Liu, Huanming Yang, Honghai Dai, Dan Wang, Wei Jin, Feng Tang, Qiang Gao, Hang Xun, Yuwei Wang, Lianghua Qi, Xianghua Yue, Shuyan Lin, **Lianfeng Gu**, Lubin Li, Tiansheng Zhu, Qiang Wei, Zhen Su, Tarmeze Bin Wan Ariffin Wan, Daniel A. Ofori, George Mbeva Muthike, Yigardu Mulatu Mengesha, Roberto Magno de Castro e Silva, Antonio Ludovico Beraldo, Zhimin Gao#, Xin Liu#, Zehui Jiang#. Announcing the Genome Atlas of Bamboo and Rattan (GABR) project: promoting research in evolution and in economically and ecologically beneficial plants. _**Gigascience**_.
[DOI:10.1093/gigascience/gix046 ](https://academic.oup.com/gigascience/article-lookup/doi/10.1093/gigascience/gix046)

Zhang, H., Lin, C. and **Gu, L #**. (2017). Light Regulation of Alternative Pre-mRNA Splicing in Plants. _**Photochem Photobiol**_.
[DOI:10.1111/php.12680.](http://onlinelibrary.wiley.com/doi/10.1111/php.12680/abstract)


### **Papers 2016(*first-author   †corresponding author)**

Wang Q*, Zuo Z*, Wang X*,** Gu L**, Yoshizumi T,Yang,Z H,Liu Q,Liu W,Han Y,Liu B, Wohlschlegel J A, Matsui M, Oka Y and Lin C.(2016). Photoactivation and inactivation mechanisms of Arabidopsis cryptochrome2.  _**Science**_.Vol. 354, Issue 6310, pp.343-347.
[DOI:10.1126/science.aaf9030](http://science.sciencemag.org/content/354/6310/343)

Yan H, Zhang H, Wang Q, Jian H, Qiu X, Baudino S, Just J, Raymond O, **Gu L**, Jihua W, Bendahmane M and Tang K (2016). The Rosa chinensis cv. Viridiflora phyllody phenotype is associated with misexpression of flower organ identity genes. _**Front. Plant Sci**_. 7:996.
[DOI:10.3389/fpls.2016.00996](http://journal.frontiersin.org/article/10.3389/fpls.2016.00996/full)

Li S, Zhang G, Zhang X, Yuan J, Deng C, **Gu L†**, Gao W†. DPTEdb, an integrative database of transposable elements in dioecious plants. _**Database**_ 2016
[DOI:10.1093/database/baw078](http://database.oxfordjournals.org/content/2016/baw078.abstract?sid=bf8d7f97-177d-40b6-ab95-149898bbf618)

Li C, **Gu L**, Gao L, Chen C, Wei C, Qiu Q, Chien C, Wang S, Jiang L, Chen C, Yang S, Nguyen V, Snyder M, Burlingame A, Kohalmi S, Huang S, Cao X, Wang Z, Wu K, Chen X, Cui Y. Concerted genomic targeting of histone H3 lysine 27 demethylase REF6 and chromatin-remodeling ATPase BRM in Arabidopsis. _**Nature genetics**_. (2016)
[DOI:10.1038/ng.3555](https://www.nature.com/articles/ng.3555#citeas)


Cui X*, Lu F*, Qiu Q*, Zhou B*, **Gu L**, Zhang S, Kang Y, Cui X, Ma X, Yao Q, Ma J, Zhang X, Cao X. REF6 recognizes a specific DNA sequence to demethylate H3K27me3 and regulate organ boundary formation in Arabidopsis. _**Nature genetics**_. (2016)
[DOI:10.1038/ng.3556](https://www.nature.com/articles/ng.3556)
	
	
X*, Lu T*, Wang L*, **Gu L**, Sun J, Kong X, Liu C, Cao X. Recruitment of the NineTeen Complex to the activated spliceosome requires AtPRMT5. _**PNAS**_ 2016
[DOI:10.1073/pnas.1522458113](http://www.pnas.org/content/113/19/5447.long)


Zhang S, Xu M, Qiu Z, Wang K, Du Y, **Gu L†**, Cui X†. Spatiotemporal transcriptome provides insights into early fruit development of Solanum lycopersicum, _**Scientific reports**_.
[DOI:10.1038/srep23173](http://www.nature.com/articles/srep23173)

Wu, Z*, Zhu D*, Lin X*, Miao L*, **Gu L**, Deng X, Yang Q, Sun K, Zhu D, Cao X, TSUGE T, Dean C, Aoyama T, Gu H, Qu L. RNA-binding proteins At RZ-1B and At RZ-1C play a critical role in regulation of pre-mRNA splicing and gene expression during Arabidopsis development. _**The Plant Cell**_ (2016): TPC2015-00949
[DOI:10.1105/tpc.15.00949](https://academic.oup.com/plcell/article/28/1/55/6098209)


### **Papers before 2016**
			<H3><li>Zhang Y.,*<strong> Gu, L.,* </strong>Hou, Y.*, Wang L., Deng, X., Hang, R.,Chen,D., Zhang,X., Zhang, Y., Liu, C., and Cao, X. 2015 Integrative genome-wide analysis reveals HLP1, a novel RNA-binding protein， regulates plant flowering by targeted alternative polyadenylation .Cell Research. 2015 Jun 23. doi: 10.1038/cr.2015.77 PMID: 26099751</li> </H3>

			<H3><li>Wei L,* <strong>Gu L,*</strong> Song X*, Cui X, Lu Z, Zhou M, Wang L, Hu F, Zhai J, Meyer B, Cao X. (2014) Dicer-like 3 produces transposable element-associated 24-nt siRNAs that control agricultural traits in rice. Proc Natl Acad Sci U S A 111: 3877-3882.</li> </H3>
<p></p>
			<H3><li>Deng, X.,* <strong> Gu, L.,* </strong> Liu, C.*, Lu, T*., Lu, F., Lu, Z., Cui, P., Pei, Y., Wang, B., Hu, S., and Cao, X. (2010). Arginine methylation mediated by the Arabidopsis homolog of PRMT5 is essential for proper pre-mRNA splicing. Proc Natl Acad Sci U S A 107, 19114-19119.</li> </H3>
<p></p>
			<H3><li>Zhou, M.*,  <strong>Gu, L.*, </strong> Li, P., Song, X., Wei, L., Chen, Z. and Cao, X. (2010) Degradome sequencing reveals endogenous small RNA targets in rice (Oryza sativa L. ssp. indica). Front. Biol., 5, 67-90.</li> </H3>
<p></p>
                        <H3><li>Wu, B., Suo, F., Lei, W., and <strong>Gu, L †. </strong>  (2014). Comprehensive Analysis of Alternative Splicing in Digitalis purpurea by Strand-Specific RNA-Seq. PLoS One 9, e106001.</li></H3>
<p></p>

			<H3><li><strong>Gu, L., </strong> and Guo, R. (2007). Genome-wide detection and analysis of alternative splicing for nucleotide binding site-leucine
-rich repeats sequences in rice. J Genet Genomics 34, 247-257.</li></H3>

			<H3><li>Wang, L., Song, X., <strong>Gu, L.,</strong> Li, X., Cao, S., Chu, C., Cui, X., Chen, X., and Cao, X. (2013). NOT2 proteins promote polymerase II-dependent transcription and interact with multiple MicroRNA biogenesis factors in Arabidopsis. Plant Cell 25, 715-727.</li></H3>
<p></p>
			<H3><li>Cui X., Jin P., Cui X., <strong>Gu, L.,</strong> Lu Z., Xue Y., Wei L., Qi J., Song X., Luo M., Gynheung An, Cao X. (2013) Control of Transposon Activity by a Histone H3K4 Demethylase in Rice. Proc Natl Acad Sci U S A. 110(5):1953-1958.</li></H3>
<p></p>
			<H3><li>Zhang L., Cheng Z., Qin R., Qiu Y., Wang J., Cui X., <strong>Gu, L.,</strong> Zhang X., Guo X., Wang D., Jiang L., Wu C., Wang H., Cao X., Wan J. (2012). Identification and Characterization of an Epi-1 allele of OsFIE1 Reveals a Regulatory Linkage between two Epigenetic Marks in Rice. Plant Cell 24, 4407-4421.</li></H3>
<p></p>
			<H3><li>Zhou H., Zhou M., Yang Y., Li J., Zhu L., Jiang D., Dong J., Liu Q., <strong>Gu L.,</strong> Zhou L., … Cao X., Zhuang C. (2014). RNAase Zs1 processes UBL40 mRNA and controls thermo-sensitive genic male sterility in rice. NATURE COMMUNICATIONS. doi:10.1038/ncomms5884</li></H3>
<p></p>





### Support or Contact

Having trouble with Pages? Check out our [documentation](https://docs.github.com/categories/github-pages-basics/) or [contact support](https://support.github.com/contact) and we’ll help you sort it out.
